app/views/layouts/gtl/_grid.html.haml

```diff
@@ -14,7 +14,8 @@
                   = check_box_tag("check_#{to_cid(@id)}", 1, false,
                   :id      => "listcheckbox",
                   :style   => "padding-bottom: 0px; margin-bottom: 0px",
-                  :onclick => "miqUpdateButtons(this,'#{button_div}');")
+                  :class   => "list-grid-checkbox",
+                  :onclick => "miqGridOnCheck(this,'#{button_div}', 'gtl_div');")
```

Comment:
Not quite.. it was in all 3 partials..

On Wed, Nov 25, 2015 at 10:27 PM, Roman Blanco <notifications@github.com>
wrote:

> In app/views/layouts/gtl/_grid.html.haml
> <https://github.com/ManageIQ/manageiq/pull/5532#discussion_r45920811>:
>
> > @@ -14,7 +14,8 @@
> >                    = check_box_tag("check_#{to_cid(@id)}", 1, false,
> >                    :id      => "listcheckbox",
> >                    :style   => "padding-bottom: 0px; margin-bottom: 0px",
> > -                  :onclick => "miqUpdateButtons(this,'#{button_div}');")
> > +                  :class   => "list-grid-checkbox",
> > +                  :onclick => "miqGridOnCheck(this,'#{button_div}', 'gtl_div');")
>
> @himdel <https://github.com/himdel> thanks, corrected [image: :blush:]
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/ManageIQ/manageiq/pull/5532/files#r45920811>.
>